quickstarts/tidbcloud/config.yml

```diff
@@ -0,0 +1,50 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: tidbcloud-quickstart
+
+# Displayed in the UI (required)
+title: TiDB Cloud
+
+# Long-form description of the quickstart (required)
+description: |-
+  ## Quickstart for TiDB Cloud monitoring
+  TiDB Cloud is a fully-managed Database-as-a-Service (DBaaS) solution for TiDB, an open-source, distributed SQL database that is designed to handle both online transaction processing (OLTP) and online analytical processing (OLAP) workloads.
+  
+  ### New Relic TiDB Cloud quickstart features
+  With monitoring statistics enabled, you can observe key metrics to analyze the performance of your cluster. 
+  
+  ### TiDBCloud Quickstart instructions
+  Check out [Metrics available to New Relic](https://docs.pingcap.com/tidbcloud/monitor-newrelic-integration#metrics-available-to-newrelic) to learn more about New Relic monitoring for TiDBCloud.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+    New Relic's TiDB Cloud quickstart provides performance insights including database time, queries per second, and resource utilization. Install the quickstart to better understand utilization of resources and monitor performance issues.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - PingCAP
+  - lilyjazz
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
```

Comment:
If you want your quickstart to appear under a certain category in add data, please add specific keywords that will map to a specific category. The terms that appear match to the displayName.
[Query](https://api.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)

Reply:
THX, I change the "keywords" to "database".